endToEndTests/test/queries/GroupByLineageOrderByCountLimit.json

```
@@ -0,0 +1,32 @@
+{
+  "testCaseName": "Group all sequences by lineage and order by count descending, with limit 4",
+  "query": {
+    "action": {
+      "type": "Aggregated",
+      "groupByFields": ["pango_lineage"],
+      "orderByFields": ["count desc"],
```

We could also think about passing the information like this:
```suggestion
      "orderByFields": [{ "fields": "count", "descending": true }]
```
With descending being false by default.

---

But sometimes the user may want to order ascending by some columns and descending by others (e.g. ascending year, descending count within year). I would not neccessarily want to bloat requests like that. It is more of a feature for advanced users where we get away with the current "easter egg"-like functionality